testing/iframe.js

```diff
@@ -152,7 +152,7 @@ export function createFixtureIframe(fixture, initialIframeHeight, opt_beforeLoad
       };
       let timeout = setTimeout(function() {
         reject(new Error('Timeout waiting for elements to start loading.'));
-      }, 2000);
+      }, process.env.TEST_TIMEOUT);
```

Comment:
It worked for me locally but I don't see precedence for this. Changed it to use `ampTestRuntimeConfig` instead.